### DISCUSSION TITLE
pull request

### DISCUSSION THREAD

**Discussion by emiliamariaa9933** [frontendbr/forum#1998] [Discussão]
Estou com dificuldades para subir meu codigo para o git. Consigo fazer o pull, mas quando chega ao repositório, pede pra fazer o pull request. Eu clico no botão e não traz nada pra comparar. Minhas pastas nao aparecem no repositorio. É
![erro_git](https://user-images.githubusercontent.com/61381569/106317452-19d00880-624d-11eb-9349-d067f870ea1f.jpg)
 o primeiro commit

**Comment by leandroruel** [comment]
primeiro, certifique-se de que tenha seguido esses passos:
no seu terminal do git bash digite:
1. `git add .`
2. `git commit -m "first commit"`
3. `git push -u origin master` (se é a primeira vez que esta dando push se não apenas `git push`)
